tests/tests/test_timetable.py

```diff
@@ -81,14 +93,15 @@ def test_get_timetable(
     timetable_id = timetable_id or small_scenario.timetable
 
     response = requests.get(f"{service_url}timetable/{timetable_id}/")
+    print(response.json())
```

Comment:
```suggestion
```

Reply:
Is there a reason why it's resolved? I would have commented the same thing

Reply:
I fixed it in https://github.com/osrd-project/osrd/commit/279df9656859a3fb90e0e07cf12860a0d5a9cf26 but somehow the commit doesn't show here 🤔

Reply:
Well, now the commits has decided to show himself so I'm resolving 😁